src/controllers/Document.js

```diff
@@ -195,11 +195,11 @@ class DocumentController extends BaseController {
       action: 'search',
     };
 
-    for (const opt of ['from', 'size', 'scroll']) {
+    for (const opt of ['from', 'size', 'scroll', 'verb']) {
```

Comment:
You don't need to pass the verb as an argument since the `verb` option on query will change the http verb
```suggestion
    for (const opt of ['from', 'size', 'scroll']) {
```